core/scss/utilities/functions/flex/index.scss

```diff
@@ -0,0 +1,16 @@
+@charset "UTF-8";
+
+//
+// Flex Grid
+//
+// Style guide: Functions.Flex
+//
+
+@import
+  'breakpoint-insert',
```

Comment:
@josephgknox 
Should we move the breakpoint functions to a breakpoint folder?

Reply:
Yes.

Reply:
Done, done.